sidebars.js

```diff
@@ -281,6 +281,7 @@ module.exports = {
         type: "category",
         label: "Stacks Plugins",
         items: [
+          "nx/create-stacks-workspace/ensono-stacks-create-stacks-workspace",
```

Comment:
Could we also update the main /docs/nx/nx_stacks#option-2-create-stacks-workspace-generator section to point to this page?

Reply:
Added link